packages/unmock-core/src/__tests__/runner.test.ts

```diff
@@ -0,0 +1,93 @@
+import { buildFetch, Fetch } from "unmock-fetch";
+import { Service, transform, UnmockPackage } from "..";
+import { Backend } from "../backend";
+import { IInterceptorOptions } from "../interceptor";
+import { testServiceDefLoader } from "./utils";
+
+const { withCodes } = transform;
+
+let fetch: Fetch;
+
+const interceptorFactory = (opts: IInterceptorOptions) => {
+  fetch = buildFetch(opts.onSerializedRequest);
+  return {
+    disable() {
+      // @ts-ignore
+      fetch = undefined;
+    },
+  };
+};
+
+const backend = new Backend({
+  interceptorFactory,
+  serviceDefLoader: testServiceDefLoader,
+});
+
+const unmock = new UnmockPackage(backend);
+
+const runner = unmock.runner.bind(unmock);
```

Comment:
Does one have to use `unmock.runner.bind(unmock)` now? Or is it simply because we're testing this within the `core` package?

Reply:
Used simply `unmock.runner` instead.